cypress_shared/components/personDetails.ts

```diff
@@ -0,0 +1,27 @@
+import type { Person } from '@approved-premises/api'
+import { DateFormats } from '../../server/utils/dateUtils'
+import Component from './component'
+
+export default class PersonDetailsComponent extends Component {
+  constructor(private readonly person: Person) {
+    super()
+  }
+
+  shouldShowPersonDetails(): void {
+    cy.get('dl[data-cy-person-info]').within(() => {
+      this.assertDefinition('Name', this.person.name)
+      this.assertDefinition('CRN', this.person.crn)
+      this.assertDefinition('Date of birth', DateFormats.isoDateToUIDate(this.person.dateOfBirth, { format: 'short' }))
+      this.assertDefinition('NOMS number', this.person.nomsNumber)
```

Comment:
I've learnt today that we can no longer assume people have noms numbers and the lack of one should not prevent a referral. I heard from Adam that CAS1 are refactoring things to make this and other bits of oasys risk data optional to work around this.

Reply:
Reckon that's a separate PR? Will merge this in as is, and flag in backlog refinement that making NOMS optional is coming up